### PR TITLE
[FIX] Workspace input without i18n

### PR DESCRIPTION
**Pull request by sumukhah**
* Strings are added for translation. Provided a default value if the translation fails.

fixes: #2620

## Issue(s)	

Some strings were not added for translation at the login screen.

## How to test or reproduce

Add translated string to respective files (such as app/i18n/locales/ar.js, app/i18n/locales/fr.js) and visit the login screen. The translated strings should be visible.

## Screenshots
Screenshot 2020-12-04 at 7 23 22 PM

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

Translated strings are added only to the English language. Still need to add translated strings to other languages.

### CODE REVIEW

File: app/views/NewServerView/ServerInput/index.js [hunk 1]
```diff
@@ -42,8 +43,8 @@ const ServerInput = ({
 	return (
 		<View style={styles.container}>
 			<TextInput
-				label='Enter workspace URL'
-				placeholder='Ex. your-company.rocket.chat'
+				label={I18n.t('Enter_workspace_URL', { defaultValue: 'Enter workspace URL' })}
```
Comment (diegolmello) [inline app/views/NewServerView/ServerInput/index.js:46]: No need for `defaultValue`.
The app always falls back to `en`.

**Review by diegolmello — CHANGES_REQUESTED** [review]

File: app/views/NewServerView/ServerInput/index.js (hunk 1 above)
Comment (sumukhah) [reply app/views/NewServerView/ServerInput/index.js]: Removed the defaultValue in this commit.

File: app/i18n/locales/en.js [hunk 2]
```diff
@@ -704,5 +704,7 @@ export default {
 	The_user_will_be_removed_from_s: 'The user will be removed from {{s}}',
 	Yes_remove_user: 'Yes, remove user!',
 	Direct_message: 'Direct message',
-	Message_Ignored: 'Message ignored. Tap to display it.'
+	Message_Ignored: 'Message ignored. Tap to display it.',
+	Enter_workspace_URL: 'Enter workspace URL',
+	Workspace_URL_Example: 'Ex. your-company.rocket.chat'
```
Comment (sumukhah) [inline app/i18n/locales/en.js:709]: If I provide key-value 'Ex. your-company.rocket.chat', will result in an error.
For that reason, changed it's key name to be Workspace_URL_Example

Screenshot 2020-12-07 at 5 33 02 PM

**Comment by sumukhah** [comment]
@diegolmello Committed changes.. Please review

**Review by diegolmello — APPROVED** [review]
LGTM

**Pull request merged**